content/references/rippled-api/public-rippled-methods/server-info-methods/fee.md

```diff
@@ -157,7 +157,8 @@ The response follows the [standard format][], with a successful result containin
 
 ## Possible Errors
 
-* Any of the [universal error types][].
+- Any of the [universal error types][].
+- `reportingUnsupported` - ([Reporting Mode][] servers only) This method is not available in Reporting Mode.
```

Comment:
`fee` is available in reporting mode. The reporting server will just forward the request to the p2p node. If that fails, the error will be `failedToForward`.